homework/jenya_s/homework12.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+class Cpypl:
+    def __init__(self, directory):
+        self.directory = directory
+
+    def c_ext(self):

```

Comment:
Hm, you should optimize your code, because I see a lot of similar code blocks

Reply:
fixed, please check @aodarc